packages/backend/src/server/api/endpoints/admin/show-user.ts

```diff
@@ -34,8 +35,37 @@ export default define(meta, paramDef, async (ps, me) => {
 		throw new Error('cannot show info of admin');
 	}
 
+	if (!_me.isAdmin) {
```

Comment:
I am not sure what `_me` is for in this case as I think it should be eqivalent to `me`. We might want to remove `_me` in later refactoring, so please use `me` here instead.
```suggestion
	if (!me.isAdmin) {
```

Reply:
The _me is used as it was already used in the lines before ¯\_(ツ)_/¯